docs/tutorials/additional-resources.txt

```diff
@@ -0,0 +1,121 @@
+********************
+Additional Resources
+********************
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Mongoid has an extensive ecosystem. Some of the third-party guides and
+blog posts about Mongoid, as well as libraries integrating with or built
+on top of Mongoid, are listed below.
+
+Screencasts
+===========
+
+- `RailsCasts: Mongoid (revised)
+  <https://www.youtube.com/watch?v=L0RqU2MdqXU>`_
+
+  An overview of Mongoid, by Ryan Bates including the basics 
+  of setting up an app, querying for records, adding embedded 
+  associations, overriding the id, and more.
+
+- `Ruby on Rails Web Services and Integration with MongoDB, Week 3: Mongoid
+  <https://www.youtube.com/watch?v=9LylgiMYsUM>`_
+
+  A detailed introduction to Mongoid and Ruby on Rails web services
+  
+- `Create a search bar in Rails with Mongoid
+  <https://www.youtube.com/watch?v=zusWR8jS5-A>`_
+
+  A Tutorial explaining how to implement text search with Mongoid
+
+Articles
+========
+
+- `Setting up Ruby on Rails 5 App with MongoDB in 10 Minutes
+  <https://www.botreetechnologies.com/blog/setting-up-ruby-on-rails-5-app-with-mongodb-in-10-minutes>`_
```

Comment:
I don't think this adds anything not already covered by our own quick start tutorial, and this guide uses Mongoid 6 and `bson_ext` which I don't think is needed at all.